Makefile

```diff
@@ -125,19 +121,18 @@ endif
 # Clear tests cache
 clear-test-cache:
 	go clean -testcache
-
-test: UNIT_TESTS = $(shell go list ./...  | grep -v systest)
+.PHONY: clear-test-cache
 
 test: get-libs
-	$(ULIMIT) CGO_LDFLAGS="$(CGO_TEST_LDFLAGS)" TEST_LOG_LEVEL=$(TEST_LOG_LEVEL) gotestsum -- -timeout 0 -p 1 $(UNIT_TESTS)
+	@$(ULIMIT) CGO_LDFLAGS="$(CGO_TEST_LDFLAGS)" gotestsum -- -timeout 0 -p 1 $(UNIT_TESTS)
 .PHONY: test
```

Comment:
`TEST_LOG_LEVEL` is used in tests for tweaking the log level. If you remove this, there will be no other sign that one can tweak the log level apart from the source code. How about adding a short note in the README that log level in tests can be modified by setting `TEST_LOG_LEVEL`?

Reply:
Updated README.md